e2e/dashboards-suite/dashboard-browse-nested.spec.ts

```diff
@@ -0,0 +1,147 @@
+import { e2e } from '@grafana/e2e';
+import { selectors } from '@grafana/e2e-selectors';
+
+import { makeNewDashboardRequestBody } from './utils/makeDashboard';
+
+const NUM_ROOT_FOLDERS = 60;
+const NUM_ROOT_DASHBOARDS = 60;
+const NUM_NESTED_FOLDERS = 60;
+const NUM_NESTED_DASHBOARDS = 60;
+
+// TODO enable this test when nested folders goes live
+describe.skip('Dashboard browse (nested)', () => {
+  const dashboardUIDsToCleanUp: string[] = [];
+  const folderUIDsToCleanUp: string[] = [];
+
+  // Add nested folder structure
+  before(() => {
+    e2e.flows.login('admin', 'admin');
+
+    // Add root folders
+    for (let i = 0; i < NUM_ROOT_FOLDERS; i++) {
+      e2e()
+        .request({
+          method: 'POST',
+          url: '/api/folders',
+          body: {
+            title: `Root folder ${i.toString().padStart(2, '0')}`,
+          },
+          headers: {
+            'Content-Type': 'application/json',
+          },
+        })
+        .then((response) => {
+          folderUIDsToCleanUp.push(response.body.uid);
+        });
+    }
+
+    // Add root dashboards
+    for (let i = 0; i < NUM_ROOT_DASHBOARDS; i++) {
+      e2e()
+        .request({
```

Comment:
Does this fire off all 60 requests in parallel?

Reply:
no, pretty sure these happen sequentially because `cy.request` has to wait for a server response: https://docs.cypress.io/api/commands/request#Requirements